narayana/scripts/hudson/benchmark.sh

```diff
@@ -130,7 +130,7 @@ case $# in
    for  i in "$BM4a" "$BM4b" "$BM4c" "$BM4d" "$BM4e" "$BM1" "$BM2" "$BM3" "$BM5"; do
      IFS=' ' read -a bms <<< "$i"
      cd $BMDIR
-     mvn -f "${bms[0]}/pom.xml" clean install -DskipTests # build the benchmarks
+     ./build.sh -f "${bms[0]}/pom.xml" clean install -DskipTests # build the benchmarks
```

Comment:
This means that we can only call the benchmark.sh script from top level of the repo. I checked the other scripts and they all assume it is called from the top level so it is a safe change. That said do we require that the WORKSPACE env variable points to the top level so using that would be less brittle.